pkg/pillar/cmd/volumemgr/handleverifier.go

```diff
@@ -84,6 +84,11 @@ func MaybeRemoveVerifyImageConfig(ctx *volumemgrContext, imageSha string) {
 			imageSha)
 		return
 	}
+	if m.Expired {
```

Comment:
The fact that this solves it is indicating something has a pointer to a blobStatus after having called MaybeRemoveVerifyImageConfig, and that blobStatus is stale.
I can't find such a location in the code...